docs/pds4_create_xml_index.rst

```diff
@@ -185,4 +197,9 @@ Miscellaneous
   be useful for debugging.
 
 - ``--config-file``: Specify a ``.ini``-style configuration file for further customization
-  of the extraction process.
+  of the extraction process. This file allows you to replace the field entry for any
+  element's data type. Its primary purpose is to handle nilled elements. Nilled elements
+  are elements that are intentionally omitted due to inapplicable, missing, unknown, or
+  anticipated values. The default configuration file (``pds4indextools.ini``) covers a
```

Comment:
People who installed this tool don't have access to the default file so this isn't helpful. Also saying it covers "a specific set" isn't helpful since they don't know what that set is.